### ISSUE TITLE
[Bug] Dynmap stars visibility can't be toggled

### ISSUE THREAD

**Issue by 3meraldK** [EarthMC/Issue-Tracker#1661]
### Server

Aurora

### Describe the bug

Pretty self-explanatory. This bug has been around for almost a few years, if not since the beginning of EarthMC when they introduced the dynamic map.

This unwanted behavior is caused by the JSON file called marker_[world].json, which is responsible for placing towns on map. It's accessible from here https://earthmc.net/map/aurora/standalone/MySQL_markers.php?marker=_markers_/marker_earth.json (for Aurora as of 4th April 2023). JSON files are more readable with extensions like JSON Viewer.

Expand 'sets' and you will see two objects, "townyPlugin.markerset" (used to be "towny.markerset" until 22nd March 2020) and "markers". "worldborder.markerset" doesn't matter. Problem is, when you expand both objects further, you will see "markers" object in each of them. And the wrong object contains stars, which is leading to this bug.

TLDR; Move sets['townyPlugin.markerset'].markers to sets.markers. It's provably working in my extension available on GitHub.

### To Reproduce

1. Enter any dynmap.
2. Toggle Markers in top-left corner. It is checked by default.
3. Stars visibility will not be toggled.

### Expected behavior

Stars should be able to be toggled on or off.

### Screenshots

![image](https://user-images.githubusercontent.com/48335651/229604291-fb32f328-a27c-40c2-bdb1-59ed8b17aece.png)
![image](https://user-images.githubusercontent.com/48335651/229606064-326c22bc-23b1-4335-bd3b-a961eb590afd.png)
